### PR TITLE
face icon reflects DM policy 

### LINKED ISSUE #19743: UI: DM icon should accurately reflect DM policy
Currently, the DM icon only shows whether a face is detected, which is confusing in cases such as model uncertainty.

### PR DESCRIPTION
**Pull request by ZwX1616**
#19743

### CODE REVIEW

File: selfdrive/ui/paint.cc [hunk 1]
```diff
@@ -307,7 +307,7 @@ static void ui_draw_driver_view(UIState *s) {
   const int face_size = 85;
   const int icon_x = is_rhd ? rect.right() - face_size - bdr_s * 2 : rect.x + face_size + bdr_s * 2;
   const int icon_y = rect.bottom() - face_size - bdr_s * 2.5;
-  ui_draw_circle_image(s, icon_x, icon_y, face_size, "driver_face", face_detected);
+  ui_draw_circle_image(s, icon_x, icon_y, face_size, "driver_face", s->scene.dmonitoring_state.getIsActiveMode());
```
Comment (ZwX1616) [inline selfdrive/ui/paint.cc:310]: in driverview the box surrounding the face will still be drawn regardless

Comment (adeebshihadeh) [reply selfdrive/ui/paint.cc:310]: sounds good

Comment (adeebshihadeh) [reply selfdrive/ui/paint.cc:310]: hmm, actually let's make a TODO and give some indicator of whether it's active in driver view

Comment (ZwX1616) [reply selfdrive/ui/paint.cc:310]: the icon would tell the activeness in driverview also

File: selfdrive/monitoring/dmonitoringd.py [hunk 2]
```diff
@@ -76,6 +76,7 @@ def dmonitoringd_thread(sm=None, pm=None):
       "isLowStd": driver_status.pose.low_std,
       "hiStdCount": driver_status.hi_stds,
       "isPreview": offroad,
+      "isActive": driver_status.active_monitoring_mode,
```
Comment (adeebshihadeh) [inline selfdrive/monitoring/dmonitoringd.py:79]:
```suggestion
      "isActiveMode": driver_status.active_monitoring_mode,
```

**Pull request merged**